docs-ref-conceptual/use-cli-effectively.md

```diff
@@ -86,13 +90,7 @@ When you work with Azure CLI commands, be aware of how your shell uses quotation
 > [!NOTE]
 > Due to a known issue in PowerShell, some extra escaping rules apply. For more information, see [Quoting issues with PowerShell](https://github.com/Azure/azure-cli/blob/dev/doc/quoting-issues-with-powershell.md).
 
-### Using hyphen characters in values
-
-If a value begins with a hyphen, Azure CLI tries to parse it as a parameter name. To parse it as value, use quotation marks: `--password="-VerySecret"`.
-
-### Using quotation marks in values
-
-If you provide a parameter a value that contains whitespace, wrap it in quotation marks. Keep the following tips in mind:
+If you provide an argument that contains whitespace, wrap it in quotation marks. Keep the following tips in mind:
```

Comment:
Change `value` back to `argument` as CLI uses the word `argument`:

```
> az group show -h

Command
    az group show : Gets a resource group.

Arguments
    --name --resource-group -g -n [Required] : Name of resource group. You can configure the default
                                               group using `az configure --defaults group=<name>`.
```

`value` is not a CLI standard.